src/ui/groups/detail/GroupDetailTable.js

```diff
@@ -31,7 +31,7 @@ class GroupDetailTable extends React.Component {
             </div>
           </div>
         </div>
-        <GroupDetailTabs references={group.references} keyList={group.referenceKeyList} />
```

Comment:
did u really want to remove this props?

Reply:
@gReis89 yes its intended. The `GroupDetailTabs` component does not use the props..